### DISCUSSION TITLE
光線追蹤

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS4719-C/Python-Community-GS4719-C#534] [General]
光線追蹤（英語：ray tracing）是[3D電腦圖形學]中的特殊演算法，追蹤光線從來源開始照射到物體上，再由物體反射的光線「路徑」，由於完整運算所有路徑十分消耗運算資源，因此現有光線追蹤技術僅運算「目所能及」的光線路徑，由於是從玩家視角開始進行運算，有時被誤解為光線追蹤是追蹤「從眼睛發出的光線」。透過這樣一項技術生成編排好的場景的數學模型顯現出來。這樣得到的結果類似於[光線投射]與[掃描線彩現]方法的結果，但是這種方法有更好的光學效果，例如對於[反射]有更準確的類比效果，並且效率非常高，所以當追求高品質的效果時經常使用這種方法。

詳情打開youtube  RTX minecraft 就可以知道有多正了

**Comment by SamTU0222** [comment]
光追真香啊！

**Comment by sean1325418** [comment]
我只知道玩遊戲時光追最好要關掉，不然電腦很快就要燒掉了

**Comment by PoooYen** [comment]
很多遊戲大作都有光追設定，但電腦硬體能不能負荷又是另一回事了

**Comment by HaoFish123** [comment]
感謝分享

**Comment by leon10456** [comment]
開光追的時候 我的電腦聽起來像飛機要起飛是正常的嗎

**Comment by HuanYuuuuuu** [comment]
https://www.4gamers.com.tw/news/detail/37691/gamer-tries-not-to-stare-at-streamers-boobs-using-eye-tracker

光線追縱也可以用在這種小遊戲上面

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來有這種功能。

**Comment by jason79461385** [comment]
光線追蹤的陰影真的很美！希望更多遊戲可以支援這項技術

**Comment by Lai27330** [comment]
像是Minecraft的光追就能讓色塊產生透明感與光線反射玻璃的變化，是個很擬真的技術。

**Comment by liangyinglly** [comment]
效能夠好的電腦才能使用光追啊

**Comment by sys511613** [comment]
光追雖然香 但最大的問題還是買不起顯卡QQ

**Comment by stan890425** [comment]
光追就是香！

**Comment by jcpan821** [comment]
謝謝分享

**Comment by alexthebigpp** [comment]
光追有夠香，拿來玩minecraft真的很爽。

**Comment by 110409538** [comment]
感謝分享!!

**Comment by bloodtrail0511** [comment]
光追真香

**Comment by etin2468** [comment]
買不到正常價格顯卡玩不起QQ

**Comment by Jimmy911218** [comment]
感謝分享

**Comment by Hchejxnc** [comment]
光追用起來真的很讚
